### ISSUE TITLE
Improve Accessibility

### ISSUE THREAD

**Issue by pgwillia** [AccessLibCon/merino#1]
[Clickable elements must be focusable and should have interactive semantics](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard#Clickable_elements_must_be_focusable_and_should_have_interactive_semantics)
